packages/fxa-auth-server/scripts/paypal-processor.ts

```diff
@@ -90,6 +90,7 @@ export async function init() {
         [program.lockName],
         lockDuration,
         async (signal: RedlockAbortSignal) => {
+          // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
We could potentially also use something like https://eslint.org/docs/latest/rules/no-unused-vars#varsignorepattern in the base .eslintrc config with some regex like `/^_$/` to have it ignore `_` variable names (although possibly a gross over-complication if we're only using this in 2-3 places).